src/vs/editor/contrib/codeAction/browser/codeActionController.ts

```diff
@@ -335,3 +362,18 @@ export class CodeActionController extends Disposable implements IEditorContribut
 		return resultActions;
 	}
 }
+
+registerThemingParticipant((theme, collector) => {
```

Comment:
Instead of using a theming participant, can you use css variables? Take a look at how `linkedEditing` implements this for example